components/eam/src/physics/cam/nucleate_ice_cam.F90

```diff
@@ -346,23 +347,25 @@ subroutine nucleate_ice_cam_init(mincld_in, bulk_scale_in)
       end if
 
       if (mode_coarse_idx > 0) then
-         call rad_cnst_get_info(0, mode_coarse_idx, nspec=nspec)
```

Comment:
I'm curious why this line got removed; what is defining nspec for the coarse mode case here? It looks like it would inherit from the last time nspec was defined in the above code, which might not be what we want.

Reply:
We remove this instance so that we can make the call within the species loop below (see line 351/352).

Reply:
Oh wait, I see what you mean now, because the removed rad_cnst_get_info() call has information on nspec, right? Let me think about this more...

Reply:
I don't think I intended to remove that line, so this is probably a bug.

Reply:
They are indeed the same based on log output from a test run, and because mode_coarse_slt_idx = mode_coarse_idx. This won't affect results for MAM4 but will be a problem for MAM7.